NodeTypes/Field/Honeypot/Honeypot.fusion

```diff
@@ -17,12 +17,20 @@ prototype(Sitegeist.PaperTiger:Field.Honeypot) < prototype(Neos.Neos:ContentComp
             condition = true
             renderer = afx`
                 <Neos.Fusion.Form:FieldContainer
-                    field.name={props.name}
                     attributes.style="display:none !important;"
                     attributes.autocomplete="off"
                     attributes.tabindex="-1"
                 >
-                    <Neos.Fusion.Form:Input/>
+                    <Neos.Fusion.Form:Input field.name={props.name + '[one]'} attributes.value="" />
+                    <Neos.Fusion.Form:Input field.name={props.name + '[two]'} attributes.value={PaperTiger.timestampWithHmac()} />
+                    <Neos.Fusion.Form:Input field.name={props.name + '[three]'} attributes.id={'field_' + props.name} attributes.value="" />
+                    <script>
+                    {'
+                        window.onload = function(e){
+                            document.getElementById("field_' + props.name + '").setAttribute("value","' + PaperTiger.timestampWithHmac() + '");
+                        }
+                    '}
+                    </script>
```

Comment:
I would prefer a slightly other script to make this even safer:

```
<script data-name={"field_" + props.name} data-value={PaperTiger.timestampWithHmac()}>
    {'
        const events = ["touchstart", "keydown", "mousemove", "touchmove"];
        const dataset = document.currentScript.dataset;

        function addEvents() {
        events.forEach(event => {
            document.addEventListener(event, insertSpamProtectionValue, true);
        });
        }

        function removeEvents() {
        events.forEach(event => {
            document.removeEventListener(event, insertSpamProtectionValue, true);
        });
        }

        function insertSpamProtectionValue() {
            const field = document.getElementById(dataset.name);
            const value = dataset.value;
            if (field && value) {
                field.value = value;
                removeEvents();
            }
        }

        addEvents();
    '}
</script>
```

Minified this would be:

```
<script data-name={"field_" + props.name} data-value={PaperTiger.timestampWithHmac()}>{'(()=>{var o=["touchstart","keydown","mousemove","touchmove"],n=document.currentScript.dataset;function u(){o.forEach(e=>{document.addEventListener(e,c,!0)})}function a(){o.forEach(e=>{document.removeEventListener(e,c,!0)})}function c(){let e=document.getElementById(n.name),t=n.value;e&&t&&(e.value=t,a())}u();})();'}</script>
```

Like that, the value will be filled on the first `touchstart`, `keydown`, `mousemove` or `touchmove` event

Reply:
I like the pattern with a script that is customized via dataset